security/security-spring/src/test/java/org/cbioportal/security/spring/authentication/token/oauth2/OAuth2TokenAuthenticationProviderTestConfiguration.java

```diff
@@ -64,6 +64,7 @@ public OAuth2TokenRefreshRestTemplate tokenRefreshRestTemplate() {
                         .append("]")
                     .append("}")
                 .append("}")
+                .append(",\"sub\": \"username\"")
```

Comment:
The section deals with the expected response of the IDP after the refresh token (opaque or JWT) has been submitted. What is the reason that you removed this response element here? Can it be retained?

Reply:
Nothing is removed. added `sub` to response token

Reply:
Ah sorry, I have it the wrong way around. Consider not mentioned :)